### PR TITLE
feat: Login page added

### PR DESCRIPTION
**Pull request by Manishsharma203**
for preview refer link below:
`https://sumuzm.csb.app/pages/login.html`

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,61 @@
...
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cruxui-lib.netlify.app//Components/main.css"
```
Comment (shubhmehra) [inline index.html:8]: there is // in your link..
is it working properly?
if not necessary please remove it!

File: index.html [hunk 2]
```diff
@@ -1,11 +1,61 @@
...
+                    <input id="submenu-input" type="checkbox" />
+                    <label for="submenu-input" class="submenu-text">
+                      Submenu
+                    </label>
+                    <ul>
+                      <li>Option 1</li>
+                      <li>Option 12</li>
+                      <li>Option 123</li>
+                    </ul>
+                  </div> -->
```
Comment (shubhmehra) [inline index.html:36]: please remove this unused lines of comments

File: index.html [hunk 3]
```diff
@@ -1,11 +1,61 @@
...
+      href="https://cruxui-lib.netlify.app//Components/main.css"
+    />
+    <link rel="stylesheet" href="style.css">
+    <title>CALYX-ECOMM</title>
+  </head>
+  <body>
+    <div class="rootContainer">
+      <div class="navbar bg-primary p-2" style="z-index: 100">
+        <input id="menu-icon" type="checkbox" />
+        <label class="menu-label" for="menu-icon"></label>
```
Comment (shubhmehra) [inline index.html:17]: you have added label but I don't think it is displaying any thing in UI..
So please remove it if it is not useful!

File: pages/cart.html [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+        </div>
+      </div>
+      <!-- Cart -->
+      <div class="grid-col gap-2 p-4">
+        <div class="col-4 col-4-md col-6-sm">
+          <div class="card card-horizontal full-width whishlist-card p-4-y">
+            <div class="card-img">
+              <img
+                src="../assets/Photo by Brian Hall on Unsplash.jpg"
+                alt="img"
```
Comment (shubhmehra) [inline pages/cart.html:68]: please provide valid alt messages for better accessibility!
It is already in in  tag then no need to add that text in alt.
you can put text like this --->"Brian hall"

File: pages/cart.html [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+          </button>
+        </div>
+      </div>
+      <!-- Cart -->
+      <div class="grid-col gap-2 p-4">
+        <div class="col-4 col-4-md col-6-sm">
+          <div class="card card-horizontal full-width whishlist-card p-4-y">
+            <div class="card-img">
+              <img
+                src="../assets/Photo by Brian Hall on Unsplash.jpg"
```
Comment (shubhmehra) [inline pages/cart.html:67]: Avoid putting spaces in names of Files/Folders
you can use underscore like this --> brian_hall.jpg

File: pages/cart.html (hunk 4 above)
Comment (shubhmehra) [reply pages/cart.html:68]: same update where applies

File: pages/sign-up.html [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+      <div class="signUp-box border-secondary box-3d p-4">
+        <div class="h5 color-secondary p-2-y">
+          Sign Up to become a Calyx user.
+        </div>
+        <label class="input-label">Email Id</label>
+        <div class="input-group">
+          <input
+            class="text-input"
+            type="text"
+            placeholder="  Enter email id..."
```
Comment (shubhmehra) [inline pages/sign-up.html:36]: input type will be email for email id input

File: pages/sign-up.html [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+        <div class="input-group">
+          <input
+            class="text-input"
+            type="text"
+            placeholder="  Enter second name..."
+          />
+        </div>
+        <label class="input-label">Enter Password</label>
+        <div class="input-group">
+          <input class="text-input" id="password-input" type="text" />
```
Comment (shubhmehra) [inline pages/sign-up.html:57]: input type is password for password inputs.
also you can add placeholder

**Review by shubhmehra — APPROVED** [review]
Added some comments

File: index.html (hunk 1 above)
Comment (Manishsharma203) [reply index.html:8]: yes its working properly

File: index.html (hunk 2 above)
Comment (Manishsharma203) [reply index.html:36]: will do

File: index.html (hunk 3 above)
Comment (Manishsharma203) [reply index.html:17]: Its functionality is critical for collapsible navbar behavior. Ui is managed by css

File: pages/cart.html (hunk 4 above)
Comment (Manishsharma203) [reply pages/cart.html:68]: sure

File: pages/cart.html (hunk 5 above)
Comment (Manishsharma203) [reply pages/cart.html:67]: will change these

File: pages/sign-up.html (hunk 6 above)
Comment (Manishsharma203) [reply pages/sign-up.html:36]: the type is being controlled by JS

File: index.html (hunk 3 above)
Comment (shubhmehra) [reply index.html:17]: ok

**Pull request merged**